README.md

```diff
@@ -62,12 +54,6 @@ Feel free to check the [issues page](../../issues/).
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
 ## 📝 License
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] When I click the link to the `MIT.md` file that you added, I get this 404 `error`:

![00Mit](https://user-images.githubusercontent.com/79058364/182822618-fdfb2523-cb23-4060-9f72-75ec8c29e4cd.png)

- Please update your Readme, it will be nice to add and update this section(`License`) with the type of License and a link to the license file, as this will make your readme more informative and professional. Checkout this [template](https://github.com/microverseinc/readme-template) for more clarifications.

Reply:
Hello @BenMukebo yeah the link is broken in this branch because I added the license file when I created the repo. It should work when I merge this branch to dev.

Reply:
![Captura de Pantalla 2022-10-17 a la(s) 3 58 47 p  m](https://user-images.githubusercontent.com/86022690/196282076-a6bba646-ae16-4a8b-90c7-0381b1de7430.png)